packages/camera/camera/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
 * Updates example to dispose animation controllers and curved animations.
+* Adds API support query for image streaming.
```

Comment:
Since this adds a new method (`supportsImageStreaming()`), this should be a minor bump to `0.11.1`. Also, this should also be moved to be listed first since its the meaningful change of the version bump.

Reply:
Changed, thanks.